### PR TITLE
Migrate certificate functions

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Carbon.Security/en-US/about_Carbon.Security.help.txt [hunk 1]
```diff
@@ -2,7 +2,16 @@ TOPIC
     about_Carbon.Security
 
 SHORT DESCRIPTION
-    Carbon.Security is a PowerShell module for...
+    Carbon.Security is a PowerShell module that:
```
Comment (DecoyJoe) [inline Carbon.Security/en-US/about_Carbon.Security.help.txt:5]: Should be `Carbon.Cryptography`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Overview
 
-The "Carbon.Security" module...
+The "Carbon.Security" module has functions that help manage X509 certificates and working with secure strings.
```
Comment (DecoyJoe) [inline README.md:3]: Should be `Carbon.Cryptography`

**Review by DecoyJoe — CHANGES_REQUESTED** [review]

File: Carbon.Security/en-US/about_Carbon.Security.help.txt (hunk 1 above)
Comment (splatteredbits) [reply Carbon.Security/en-US/about_Carbon.Security.help.txt:5]: Rename is coming in a pending PR.

File: Carbon.Security/Functions/Get-CCertificate.ps1 [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    #>
+    [CmdletBinding(DefaultParameterSetName='ByPath')]
+    [OutputType([Security.Cryptography.X509Certificates.X509Certificate2])]
+    param(
+        [Parameter(Mandatory, ParameterSetName='ByPath', Position=0)]
+        # The path to the certificate. Can be a file system path or a certificate path, e.g. `cert:\`. Wildcards supported.
+        [String]$Path,
+        
+        [Parameter(ParameterSetName='ByPath')]
+        # The password to the certificate. Can be plaintext or a [SecureString](http://msdn.microsoft.com/en-us/library/system.securestring.aspx).
```
[the remark this answers is not in the document]
Comment (splatteredbits) [reply Carbon.Security/Functions/Get-CCertificate.ps1]: Fixed.

**Review by DecoyJoe — APPROVED** [review]

**Pull request merged**